playground/mocks/data/idp/idx/authenticator-verification-select-authenticator-without-signed-nonce.json

```diff
@@ -0,0 +1,193 @@
+{
+  "stateHandle": "02im-3M2f6UXHgNfS7Ns7C85EKHzGaKw0u1CC4p9_r",
+  "version": "1.0.0",
+  "expiresAt": "2020-01-13T21:14:37.000Z",
+  "intent": "LOGIN",
+  "remediation": {
+    "type": "array",
+    "value": [
+      {
+        "rel": [
+          "create-form"
+        ],
+        "name": "select-authenticator-authenticate",
+        "href": "http://localhost:3000/idp/idx/challenge",
+        "method": "POST",
+        "accepts": "application/vnd.okta.v1+json",
+        "value": [
+          {
+            "name": "authenticator",
+            "type": "object",
+            "options": [
+              {
+                "label": "",
+                "value": {
+                  "form": {
+                    "value": [
+                      {
+                        "name": "id",
+                        "value": "auteq0lLiL9o1cYoN0g4",
+                        "required": true,
+                        "mutable": false
+                      },
+                      {
+                        "name": "methodType",
+                        "type": "string",
+                        "required": false,
+                        "mutable": true,
+                        "options": [
+                          {
```

Comment:
@ManuMalhotra-okta  if the OV enrollments does not belong to the OS that user is logging in from we should not have signed_nonce here even if there are signed_nonce enrollments for the user. Agree ?

Reply:
Yes the methodType-> options will not have the `signed_nonce` in the list.